tools/telemetry-generator/src/handlers/appInsightsCustomDataTestHandler.ts

```diff
@@ -0,0 +1,42 @@
+/*!
```

Comment:
Thoughts on naming this to `customDataHandler.ts`?

Reply:
See some of the other comments, we need both an app-insights version and a non-app-insights version. The non-app-insights one is the one that could be named `customBenchmarkHandler.ts` (note Benchmark VS Data, like I mentioned in another comment as well) to match the rest.